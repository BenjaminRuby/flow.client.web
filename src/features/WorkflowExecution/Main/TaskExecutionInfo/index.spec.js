import React from "react";
import StepSideInfo from "./index";

const props = {
  step: {
    taskName: "test",
    flowTaskStatus: "completed",
    startTime: 1540912389131,
    duration: 2178
  },
  task: {
    taskId: "2",
    taskName: "test task"
  },
  flowActivityId: "1"
};

describe("StepSideInfo --- Snapshot", () => {
  it("Capturing Snapshot of StepSideInfo", () => {
    const { baseElement } = rtlRender(<StepSideInfo {...props} />);
    expect(baseElement).toMatchSnapshot();
  });
});
