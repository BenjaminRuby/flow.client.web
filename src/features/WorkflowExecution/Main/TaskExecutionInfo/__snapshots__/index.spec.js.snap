// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StepSideInfo --- Snapshot Capturing Snapshot of StepSideInfo 1`] = `
<div
  className="c-step-side-info"
>
  <div
    className="s-step-side-info-title"
  >
    test task
  </div>
  <div
    className="c-step-side-info-details"
  >
    <div
      className="b-step-side-info-field"
    >
      <div
        className="b-step-side-info-field__key"
      >
        Type
      </div>
      <div
        className="b-step-side-info-field__value"
      >
        Task
      </div>
    </div>
    <div
      className="b-step-side-info-field"
    >
      <div
        className="b-step-side-info-field__key"
      >
        Status
      </div>
      <div
        className="b-step-side-info-field__value"
      >
        <img
          alt="Status"
        />
      </div>
    </div>
    <div
      className="b-step-side-info-field"
    >
      <div
        className="b-step-side-info-field__key"
      >
        Start Time
      </div>
      <div
        className="b-step-side-info-field__value"
      >
        January 1st 2019, 12:00:00 am
      </div>
    </div>
    <div
      className="b-step-side-info-field"
    >
      <div
        className="b-step-side-info-field__key"
      >
        Duration
      </div>
      <div
        className="b-step-side-info-field__value"
      >
        ---
      </div>
    </div>
    <div
      className="b-step-side-info-field"
    >
      <div
        className="b-step-side-info-field__key"
      >
        Log
      </div>
      <div
        className="b-step-side-info-field__value"
      >
        <TaskExecutionLog
          flowActivityId="1"
          flowTaskName="test task"
        />
      </div>
    </div>
  </div>
</div>
`;
